src/utils/migrations/0017_auto_20200721_1421.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-07-21 12:07
+from __future__ import unicode_literals
+import re
+
+from django.db import migrations, transaction
+
+def parse_email_addresses(entry):
+    match = re.findall(r'[\w\.-]+@[\w\.-]+\.\w+', entry.to)
```

Comment:
The good old problem of matching all email addresses by regex...
I'd just add a `\+` to the allowed characters in the local part